docs/_docs/monitoring-metrics/performance-statistics.adoc

```diff
@@ -80,9 +80,41 @@ You can manage the performance statistics collection via the `PerformanceStatist
 |Method | Description
 |start() | Start collecting performance statistics in the cluster.
 |stop() | Stop collecting performance statistics in the cluster.
+|rotate() | Rotate collecting performance statistics in the cluster.
 |started() | True if performance statistics collection is started.
 |===
 
+
+== Control Script
+
+You can manage the performance statistics collection via the command line interface.
```

Comment:
Lets add link to the Control script page

Reply:
done